Neos.ContentRepository.Core/Classes/Feature/NodeModification/Dto/SerializedPropertyValue.php

```diff
@@ -44,7 +60,7 @@ public static function fromArray(array $valueAndType): self
             throw new \InvalidArgumentException('Missing array key "type"', 1546524609);
         }
 
-        return new self($valueAndType['value'], $valueAndType['type']);
+        return self::create($valueAndType['value'], $valueAndType['type']);
```

Comment:
...adjust this accordingly:
```suggestion
        return $valueAndType['value'] === null ? UnsetPropertyValue::get() : self::create($valueAndType['value'], $valueAndType['type']);
```